bin/limitd

```diff
@@ -3,11 +3,31 @@
 process.title = 'limitd server';
 
 const config = require('../conf');
+const yaml = require('js-yaml');
+
+const agent = require('../lib/agent');
+const logger = agent.logger;
+const metrics = agent.metrics;
+const configurator = require('../lib/configurator');
+
 const LimitdServer = require('../server');
 const server = new LimitdServer(config);
 
 server.start();
 
+server.on('started', () => {
+  if (config.configurator) {
+    config.configurator.metrics = metrics;
+    config.configurator.logger = logger;
```

Comment:
Any reason not to pass the agent itself?

Also, the instrumentation agent is a singleton so we can just require auth0-instrumentation again and use it anywhere (if it was initialized before).